.github/BOTMETA.yml

```
@@ -1101,7 +1101,7 @@ files:
   $modules/web_infrastructure/apache2_mod_proxy.py:
     maintainers: oboukili
   $modules/web_infrastructure/apache2_module.py:
-    maintainers: berendt n0trax robinro
+    maintainers: berendt n0trax
```

Instead, add yourself to the ignore list here:
```suggestion
    maintainers: berendt n0trax
    ignore: robinro
```